client/modules/accounts/templates/profile/profile.js

```diff
@@ -71,5 +71,9 @@ Template.accountProfile.helpers({
 
   isMarketplaceGuest: function () {
     return (Reaction.hasMarketplaceAccess("guest") && !Reaction.hasAdminAccess());
+  },
+
+  isMarketplaceSeller: function () {
+    return (Reaction.hasMarketplaceAccess() && !Reaction.hasOwnerAccess());
```

Comment:
For my knowledge more than anything else, what's the reasoning behind having custom marketplace permissions checkers rather than using the `hasPermission` or `userIsInRole`?